### DISCUSSION TITLE
Can you view more than one chunk of text at a time?

### DISCUSSION THREAD

**Discussion by Wysardry** [vkbo/novelWriter#2036] [Q&A]
Is there currently a way of viewing multiple chunks of text at a time? For example, viewing a scene and all child sections as if they were one continuous file?

In Scrivener, this would be called "scrivenings mode".

**Comment by vkbo** [comment]
There is an old feature request for multi-document editing and viewing: #102

In principle, multi-document viewing is simple. It is what the preview in the manuscript tool does, and adding this feature to the viewer is relatively straight forward.

For editing, it is not trivial, as discussed in that feature request.

**Reply by Wysardry** [reply]
Thanks. I'm not overly concerned about not being able to edit multiple documents at the same time, especially if that slowed the program down.

Being able to view multiple documents to check that all the pieces fit together correctly would be a big help though.

Using the build preview to check this seems a little long winded, as it seems you have to set the parts you don't want to see as inactive if you don't want to preview the whole novel.

**Reply by vkbo** [reply]
I didn't mean that the manuscript preview should be used for this. I was just saying that novelWriter is already capable of generating these views, so it's not a big deal to add the feature to the regular viewer.

Feel free to make a feature request for it in Issues since #102 is mainly about multi-document editing. I can probably add multi-document view already in the next release. I can at least add a simple implementation where you can right click a document and select "View Document Tree" or something similar. Allowing multi-select and "View Documents" should also be straightforward. The preview feature will append any list of documents, so it isn't particularly complicated. I just need to figure out how to handle some of the other features that assume only one document is being viewed at a time.

**Reply by Wysardry** [reply]
I just meant that the current build preview would be cumbersome if added to the regular viewer as-is.

I've added feature request [#2037](https://github.com/vkbo/novelWriter/issues/2037).

**Reply by vkbo** [reply]
Ah, no, that's not what I meant either. I just meant how the preview document is generated. It is iteratively generated from a list of documents, and that can easily be done in the viewer too. I didn't mean to include the full Build Settings filtering options.
